src/encoded/static/components/matrix_reference_epigenome.js

```diff
@@ -415,6 +411,92 @@ MatrixHeader.contextTypes = {
     navigate: PropTypes.func,
 };
 
+const ClickableThumbnail = (props) => {
+    const expandedThumbnail = props.expandedThumbnail;
+    const expandThumbnail = props.expandThumbnail;
+    return (
+        <button
+            className={`body-image-thumbnail ${expandedThumbnail ? 'expanded' : 'collapsed'}`}
+            onClick={() => expandThumbnail()}
+        >
+            <div className="body-map-expander">Filter results by body diagram</div>
+            {svgIcon('expandArrows')}
+            <BodyDiagram />
+            <div className="body-list body-list-narrow">
+                <ul className="body-list-inner">
+                    {Object.keys(CellsList).map(image =>
+                        <div
+                            className={`body-inset ${image}`}
+                            id={image}
+                            key={image}
+                        >
+                            <img className="active-image" src={`/static/img/bodyMap/insetSVGs/${image.replace(' ', '_')}.svg`} alt={image} />
```

Comment:
hardcoding part of the url path seems like a code smell I think. I feel there should be some pre-existing constant or it should be made into one.